### ISSUE TITLE
pgmodeler-cli on importDatabase: The catalog query returned more than one OID!

### ISSUE THREAD

**Issue by vRoussel** [pgmodeler/pgmodeler#1726]
**Bug description**
The cli always aborts with this error when it tries to import an existing database. I don't have this issue with the GUI app. I can import the db, check for diffs, explore it, etc.

**How to reproduce**
`postgres=> create database pgmodeler_test_201122;`
`$ pgmodeler-cli -id pgmodeler_test_201122 -of test.sql -im -H localhost`

**Expected behavior**
The cli should be able to import a database with no issues.

**Info about your desktop**
 - OS: Arch linux
 - Version: Linux XXX 5.15.79-1-lts 1 SMP Wed, 16 Nov 2022 11:03:21 +0000 x86_64 GNU/Linux
 - Window manager: i3
 - pgModeler version: 1.0.0-beta - 20221001.032e35186
 - Qt version: Qt 5.15.6

**Stacktrace / Debug info**

```
pgModeler command line interface.
Version 1.0.0-beta - 20221001.032e35186 Qt 5.15.6
PostgreSQL Database Modeler Project - pgmodeler.io
Copyright 2006-2022 Raphael Araújo e Silva <raphael@pgmodeler.io>

Starting database import...
Input database: pgmodeler_test_201122@localhost

[0] src/pgmodelercliapp.cpp (1888)
  void PgModelerCliApp::importDatabase(DatabaseModel*, Connection)
    [Custom] The catalog query returned more than one OID!

[1] src/catalog.cpp (905)
  QString Catalog::getObjectOID(const QString&, ObjectType, const QString&, const QString&)
    [Custom] The catalog query returned more than one OID!
       ** Object type: database

[2] src/catalog.cpp (893)
  QString Catalog::getObjectOID(const QString&, ObjectType, const QString&, const QString&)
    [Custom] The catalog query returned more than one OID!

** pgmodeler-cli aborted due to critical error(s). **
```

**Additional info**
I think it worked with the 1.0.0-alpha.

**Comment by rkhaotix** [comment]
This issue was fixed in 1.0.0-beta1 (under development). You can try the latest commit in the branch 1.0.0-beta1.

**Comment by vRoussel** [comment]
Alright, I'm using the aur version which is based on the releases on github. I'll just wait for the next one and use the gui in the meantime. Thank you for the (very) quick answer !
